scripts/rnnlm/choose_features.py

```diff
@@ -7,6 +7,7 @@
 import argparse
 import sys
 import math
+sys.stdout = open(1, 'w', encoding='utf-8', closefd=False)
```

Comment:
I'll merge this as I assume it's right, but can you explain why this is necessary and what the default behavior is?
I would have thought that UTF-8 is automatically compatible with/convertible-to raw bytes, and that sys.stdout would by default be outputting raw bytes.  But perhaps the issue is that it's locale dependent and it might try to convert to other encodings dependent on the locale?